src/com/spikes2212/genericsubsystems/limitationFunctions/TwoLimits.java

```diff
@@ -0,0 +1,53 @@
+package com.spikes2212.genericsubsystems.limitationFunctions;
+
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import com.spikes2212.genericsubsystems.BasicSubsystem;
+
+/**
+ * This is a Function from double to boolean. This is for a
+ * {@link BasicSubsystem} that moves between two limits.
+ * 

```

Comment:
make it very clear that we use this as the function in the basicSubsystem constructor

Reply:
we should add something about it in basicSubsystem, too